releases.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: default
+title: "Releases"
+---
+
+<section class="section">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-8 mx-auto">
+        <h1 class="section-title">Release Announcements</h1>
+        <p class="lead text-muted">Latest releases and updates from the asimov ecosystem.</p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="section bg-light">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-8 mx-auto">
+        <h2 class="h3 fw-bold mb-4">Recent Releases</h2>
+        
+        <div class="alert alert-info">
+          <strong>Note:</strong> Release information is automatically tracked through GitHub. Visit each package's repository to view the latest releases and changelogs.
+        </div>
```

Comment:
Is there any way to automatically update the latest version number (and have a list of previous releases) created from github?